manage-data/data-store/data-streams/failure-store.md

```diff
@@ -0,0 +1,848 @@
+---
+applies_to:
+  stack: ga 8.19.0
```

Comment:
Can we put both 8.19 and 9.1 here? I want to make sure a user doesn't assume 9.0 > 8.19 and therefore should have this feature (even though it doesn't)